cfpb/consumerfinance.gov
# Add v1/tests to Python flake8 checks

Currently, [for historical reasons](https://github.com/cfpb/consumerfinance.gov/pull/2509), the Python files under v1/tests are excluded from our flake8 linter tests. This commit makes the necessary fixes to get these files to pass the linter, and restores v1/tests to the list of directories that will get checked in future.

## How to test this PR

`tox -e lint`


There are a lot of files changed here; the key one is `.flake8` where you can see the configuration change. The rest of the file changes are just whitespace and code formatting.

## Notes and todos

- [As of last week](https://github.com/psf/black/releases/tag/22.1.0), the [Black](https://black.readthedocs.io/en/stable/) Python auto-formatter is now stable - I know we've discussed this internally a bunch, and it might be a good time to consider adopting it to avoid the need for this kind of linting in future.

## Checklist
